Description: Modify XBMC to compile on all architectures
 Upstream's configure.in prevents building XBMC on unknown architectures,
 but build may succeed on some of them. This patch lets configure
 continue everywhere.
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/configure.in
+++ b/configure.in
@@ -683,7 +683,7 @@
   powerpc-*-linux-gnu*|powerpc-*-linux-uclibc*)
      ARCH="powerpc-linux"
      ;;
-  powerpc64-*-linux-gnu*|powerpc64-*-linux-uclibc*)
+  powerpc64-*-linux-gnu*|powerpc64-*-linux-uclibc*|ppc64-linux-gnu)
      ARCH="powerpc64-linux"
      ;;
   arm*-*-linux-gnu*|arm*-*-linux-uclibc*)
@@ -713,8 +713,13 @@
      use_wayland=no
      build_shared_lib=yes
      ;;
+  mipsel-linux-gnu)
+     ARCH="mipsel"
+     AC_SUBST(ARCH_DEFINES, "-DTARGET_POSIX -DTARGET_LINUX -D_LINUX -D_MIPSEL)
+     ;;
   *)
-     AC_MSG_ERROR(unsupported host ($host))
+     ARCH=$(dpkg-architecture -t$host -qDEB_HOST_ARCH)
+     AC_SUBST(ARCH_DEFINES, "-DTARGET_POSIX -DTARGET_LINUX -D_LINUX)
 esac
 AC_SUBST([ARCH])
 
